spec/controllers/articles_controller_spec.rb

```diff
@@ -27,20 +27,21 @@
         end
     end
 
-    describe '#update' do
-      describe 'update a record that already exists,'do
+    describe 'GET #update' do
```

Comment:
This is not a `GET` method. Change it for the appropriate one.

Read this: http://www.restapitutorial.com/lessons/httpmethods.html.